devicon.json

```
@@ -11721,6 +11600,30 @@
         "color": "#3b4b72",
         "aliases": []
     },
+    {
+        "name": "sfml",
+        "altnames": [
+            "simpleandfastmultimedia",
+            "simpleandfastmultimedialibrary"
```

Other altnames include spaces, so i suggest

```suggestion
            "Simple and Fast Multimedia",
            "Simple and Fast Multimedia Library"
```